kafka/README.md

```diff
@@ -0,0 +1,35 @@
+# Kafka
+
+The Kafka component subscribes to the Strimzi operator, which provides Kafka.  It also creates a
+Kafka instance in your project.
+
+## Folders
+There are two main folders in the Kafka component
+1. cluster: contains the subscription to the Strimzi operator
+2. kafka: contains installation of the Kafka instance
+
+## Kafka Instance
+
+The installed Kafka instance is named odh-message-bus and is intended for general use within Open Data Hub
+
+# Installation
+To install Kafka add the following to the `kfctl` yaml file.
+
+```
+  - kustomizeConfig:
+      parameters:
+      - name: namespace
```

Comment:
Do not recommend using the `namespace` - that basically hardcodes the namespace for the component

Reply:
Agree...I think this was a leftover from a bad copy/paste